bundles/http/Citrix/Zoom-Meetings/e40510b3-a760-43b4-ab16-56572dc6303b.js

```diff
@@ -75,11 +83,9 @@ async function syncMeetings(dataStore, client, users) {
 }
 //Loading the meetings details from zoom SOR to cache db(Done)
 async function syncMeetingsDetails(dataStore, client,meetingids) {
-    let dialIncounter = 0;
-    let dialInNumberscounter = 0;
     let meetingsMap = new Map();
-    for (const meetingid of meetingids) {
-        const response = await client.fetch(`meetings/${meetingid}`)
+    for (const value of meetingids) {
```

Comment:
provide meaningful constant name instead of general `value`, note `meetingids` is no longer list of ids